### ISSUE TITLE
DOB field in personal details form is required but not marked as such

### ISSUE THREAD

**Issue by emmastephenson** [CDCgov/prime-simplereport#4079]
## Area or Component
Sign up flow - Experian personal details form

## Quick Summary
The date of birth field is required but not marked as such. It's missing both the `*` and the `aria-required` label, but leaving it empty causes a form submission error.

### How to Replicate
1. Navigate to the Experian personal details form (https://dev.simplereport.gov/app/sign-up/identity-verification)
2. Tab over the elements, leaving DOB blank
3. See the error message "a valid date of birth is required"

Alternatively:
Don't click on the DOB field at all and try to submit the form without it
See the form error toast

## Accessibility Violation
Required field missing aria-required: https://cdc-devtools.dequecloud.com/rule-help/en/aria-required-missing

## Action Requested
Add aria-required to the DOB and style it as a required element.

## Screenshots (optional)
![Screen Shot 2022-07-26 at 3 37 19 PM](https://user-images.githubusercontent.com/80282552/181124145-62865140-f158-4118-81c6-701ab6e4a7b0.png)

![Screen Shot 2022-07-26 at 3 37 13 PM](https://user-images.githubusercontent.com/80282552/181124157-f0ae4ade-9df3-4892-afc2-a3d46cb39669.png)

## Original Deque Issue
https://docs.google.com/spreadsheets/d/1b6xjubAN5Uw11DV4_B5zvSu7HeDI6Z-DpV4DMcToqAY/edit#gid=543263785&range=116:116

**Comment by fzhao99** [comment]
This seems to be an issue only in dev / a misaligned deploy error. here's the page in staging / test that had the right form decorators. Also what I got locally once I pulled it down.

<img width="977" alt="Screen Shot 2022-09-15 at 10 03 37 AM" src="https://user-images.githubusercontent.com/29645040/190439505-fb704a2e-be70-4802-aef7-b015b1717589.png">
<img width="874" alt="Screen Shot 2022-09-15 at 10 02 02 AM" src="https://user-images.githubusercontent.com/29645040/190439510-b30a0a8f-9738-411e-9756-6a04f509758a.png">

I redeployed to dev just to check and it looks like that fixed it. Closing this for now based on the above investigation. 

<img width="903" alt="Screen Shot 2022-09-15 at 10 09 55 AM" src="https://user-images.githubusercontent.com/29645040/190440423-1f709b82-2c11-44d1-b209-de4a199b647f.png">
